tools/scripts/format-branch.sh

```diff
@@ -23,12 +23,23 @@ trap 'rm -f $PATCHY' EXIT
 
 git diff main -U0 --no-color >"$PATCHY"
 
-"$RELPATH"/clang-format-diff.py -regex '.*(\.h$|\.c$|\.cl$)' -i -p1 -style=file:"$RELPATH/style.GNU" <"$PATCHY"
+if [ -z "${CLANG_FORMAT_BIN}" ]; then
+  CLANG_FORMAT_BIN=clang-format
+  for i in {25..14}; do
```

Comment:
1) LLVM 14 does not work with our scripts anymore (it can't parse the `style.GNU`), 15 seems to parse it fine
2) could you add this to the other format scripts in `tools/scripts` as well ?

Reply:
Oops, didn't meant push formatter script changes in this PR.

Reply:
Reverted but will be reintroduced in a separate PR.

> LLVM 14 does not work with our scripts anymore (it can't parse the style.GNU)

Seems to be the case.

> could you add this to the other format scripts in tools/scripts as well ?

I'm working on this one.